packages/core/src/components/button/button-group.md

```diff
@@ -26,32 +26,33 @@ The component also supports all HTML `<div>` props.
 
 @## Usage with popovers
 
-__Button__ elements inside a __ButtonGroup__ can trivially be wrapped with a [__Popover__](#core/components/popover) to
-create complex toolbars.
+**Button** elements inside a **ButtonGroup** can be wrapped with a [**Popover**](#core/components/popover) to create
+complex toolbars or to provide split button functionality, allowing the action of a **Button** to be changed.
```

Comment:
I don't think "split button" is the unique thing to call out here in the use with popovers section, maybe "menu button"?

Also not sure about `allowing the action of a **Button** to be changed` - pretty much anything could be done inside the popover so it feels weird to call out a specific case here. I think it would be weird if the popover just contained options that would change the wrapped button's behavior (implying it needs to be clicked again?).

Reply:
Yeah, fair! I originally modeled this after GitHub's own merge options button, which I've seen as a pretty common dropdown use pattern in other places. Totally fine with omitting this and leaving the docs as is if the example isn't adding value.

Reply:
Reverted 77763571d4711d507d4456c23cdcf12c06107296